app/src/main/java/org/jellyfin/androidtv/integration/dream/composable/DreamContentLibraryShowcase.kt

```diff
@@ -35,7 +36,7 @@ fun DreamContentLibraryShowcase(
 			contentDescription = null,
 			alignment = Alignment.Center,
 			contentScale = ContentScale.Crop,
-			modifier = Modifier.fillMaxSize()
+			modifier = Modifier.fillMaxSize().alpha(0.7f)
```

Comment:
Why add alpha to the backdrop?

Reply:
I've done it to  "wakes up" or "dim down" switching into the screen saver, rather than have the screen at full brightness. On a real TV it feels like we're in a dedicated screensaver rather than just a library preview

It's a bit of scope creep, but I thought I'd float it whilst already doing it?

Reply:
But it is a dedicated screensaver?